plugin-openshift/che-plugin-openshift-ext-client/src/main/java/org/eclipse/che/ide/ext/openshift/client/importapp/ImportApplicationViewImpl.java

```diff
@@ -114,9 +118,14 @@ public SpanElement renderCategory(Category<BuildConfig> category) {
 
 
     @Inject
-    public ImportApplicationViewImpl(OpenshiftLocalizationConstant locale, CoreLocalizationConstant constants,
-                                     org.eclipse.che.ide.Resources resources) {
+    public ImportApplicationViewImpl(OpenshiftLocalizationConstant locale,
+                                     CoreLocalizationConstant constants,
+                                     org.eclipse.che.ide.Resources resources,
+                                     OpenshiftResources openshiftResources) {
         this.locale = locale;
+        this.openshiftResources = openshiftResources;
+
+        openshiftResources.css().ensureInjected();
```

Comment:
can remove this, we inject css in org.eclipse.che.ide.ext.openshift.client.OpenshiftExtension